app/src/main/java/com/vitaliidiadchenko/mykotkinapplication/FragmentMovieDetail.kt

```diff
@@ -53,18 +61,47 @@ class FragmentMovieDetail : Fragment() {
         listener = null
     }
 
-    private fun updateData() {
-        (recyclerView?.adapter as? ActorViewHolderAdapter)?.apply {
-            bindActors(listActors)
+    private fun setupView(movie: Movie) {
+        view?.run {
+            // set backdrop poster
+            val backdrop = findViewById<ImageView>(R.id.image_poster)
+            Glide.with(context)
+                .load(movie.backdrop)
+                .into(backdrop)
+
+            findViewById<TextView>(R.id.movieTitle).text = movie.title
+            findViewById<TextView>(R.id.tag_line).text =
+                movie.genres.joinToString(separator = ", ") { it.name }
+            findViewById<TextView>(R.id.storyline).text = movie.overview
+            findViewById<RatingBar>(R.id.ratingBar).rating = (movie.ratings / 2)
+            findViewById<TextView>(R.id.textReview).text =
+                resources.getQuantityString(R.plurals.review, 0, 0) //json doesn't have review
+            //set ageRating
+            val ageRating = findViewById<TextView>(R.id.ageRating)
+            if (movie.adult) {
+                ageRating.text = resources.getString(R.string.adult_age_rating)
+            } else {
+                ageRating.visibility = View.INVISIBLE
+            }
```

Comment:
Можно избавиться от ветки `else`, если в разметке указать `ageRating.visibility = View.INVISIBLE`. При наличии рейтинга - показывать.
```suggestion
            if (movie.adult) {
                with (findViewById<TextView>(R.id.ageRating)) {
                    text = resources.getString(R.string.adult_age_rating)
                    visibility = View.VISIBLE
                }
            }
```

Reply:
Спасибо, а я думал в котлине обязательно должно быть else

Reply:
Во всех языках программирования `else` - опциональная ветка.